specification/deviceprovisioningservices/data-plane/Microsoft.Devices/preview/2021-11-01-preview/device.json

```diff
@@ -0,0 +1,1044 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-11-01-preview",
+    "title": "ProvisioningDeviceClient",
+    "description": "API for device runtime operations with the Azure IoT Hub Device Provisioning Service"
+  },
+  "host": "global.azure-devices-provisioning.net",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/{idScope}/devices/{registrationId}/certificates/{certificateId}": {
+      "get": {
+        "tags": [
+          "GET"
+        ],
+        "summary": "Obtains the issued certificate.",
+        "operationId": "CertificateIssuance_Get",
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "idScope",
+            "in": "path",
+            "description": "The service assigned ID scope.",
+            "required": true,
+            "type": "string"
+          },
```

Comment:
To help enforce consistency, I recommend defining parameters in the `parameters` section and `$ref` them here.

Reply:
This is how swashbuckle generates it for us and we have used this style for a previous API files as well. I am inclined to leave this as is. Does that sound alright?

Reply:
One of the many sins of Swashbuckle.